options/options.html

```
@@ -294,67 +309,70 @@ <h3>介面設定</h3>
                 </button>
               </div>
             </div>
 
-            <!-- 已登入狀態（由 JS 切換顯示） -->
+            <!-- 已登入狀態 -->
             <div id="account-logged-in" style="display: none">
-              <div id="account-info" class="auth-status" aria-live="polite" aria-atomic="true" style="margin-bottom: 12px">
-                <!-- 由 JS 動態填入 email / displayName -->
+              <div class="profile-layout">
+                <div class="profile-avatar">
+                  <img id="profile-avatar-img" src="" alt="Avatar" style="display: none;" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)